### PR TITLE
Add VKs final amendments before resubmission

### PR DESCRIPTION
**Pull request by drvinceknight**

### CODE REVIEW

**Comment by drvinceknight** [comment]
Note that I have not included a compiled pdf in this PR.

**Comment by drvinceknight** [comment]
> Note that I have not included a compiled pdf in this PR.

Let me know if you'd like me to.

**Comment by Nikoleta-v3** [comment]
Nuh it's okay. I will have to change the template to fit the journal's anyway.

File: main.tex [hunk 1]
```diff
@@ -490,50 +488,47 @@ \section{Results}\label{section:results}
...
+the optimal memory-one strategy. This is evidence that in the case of multiple
+opponents, having a
 shorter memory is limiting.
 
 \begin{figure}[!htbp]
     \centering
     \includegraphics[width=.5\textwidth]{img/gambler_performance_against_mem_one.pdf}
     \caption{The ratio between the utilities of Gambler and best response memory-one
     strategy for 152 different pair of opponents.}\label{fig:utilities_gambler_mem_one}
+    % TODO Nik: a legend for the lines or a description of the lines in the
```
Comment (Nikoleta-v3) [inline main.tex]: Yes of course. I'll add those.

File: main.tex [hunk 2]
```diff
@@ -490,50 +488,47 @@ \section{Results}\label{section:results}
...
-novelty, but also a mathematical novelty of solving quadratic ratio optimisation
-problems where the quadratics are non concave. Moreover, it allowed us to 
-obtaining a condition for which in an environment of memory-one opponents
+more specifically, \textit{memory-one best responses}.
+
+\begin{itemize}
+    \item It has proven that the utility
+          of a memory-one strategy against a set of memory-one opponents can be written as a sum
+          of ratios of quadratic forms (Theorem~\ref{theorem_one}).
+    \item  There is a compact way of identifying a memory-one best response to a
```
Comment (Nikoleta-v3) [inline main.tex]:
```suggestion
    \item There is a compact way of identifying a memory-one best response to a
```

Comment (Nikoleta-v3) [inline main.tex]: Shouldn't this be?

```
more specifically, \textit{memory-one best responses}. It has proven that:

\item The utility...
\item There is a...
```

File: main.tex [hunk 3]
```diff
@@ -58,15 +58,14 @@
...
-    response} memory-one strategies as a multidimensional
-    optimisation problem. Though extortionate memory-one strategies have gained
-    much attention, we demonstrate that best response memory-one strategies do not
-    behave in an extortionate way, and moreover, for memory-one strategies to be
-    a best response to a group of opponents and themselves they need to be able to behave in a forgiving way. We
+    against single opponents. This manuscript investigates a theory of mind: \textit{best
+    response} memory-one strategies, as a multidimensional
+    optimisation problem. We add to the literature that has shown that
+    extortionate play is not always optimal by showing that optimal play is in
+    fact rarely extortionate. We
```
Comment (Nikoleta-v3) [inline main.tex]: I am not sure about the word rarely. The distribution of the SSE is skewed towards zero.

Comment (drvinceknight) [reply main.tex]: Yeah `rarely` is an exaggeration. Perhaps `that optimal play is often not extortionate`?

File: main.tex (hunk 2 above)
Comment (drvinceknight) [inline main.tex]:
```suggestion
more specifically, \textit{memory-one best responses}. It has shown that:

\begin{itemize}
    \item The utility
          of a memory-one strategy against a set of memory-one opponents can be written as a sum
          of ratios of quadratic forms (Theorem~\ref{theorem_one}).
    \item  There is a compact way of identifying a memory-one best response to a
```

File: main.tex (hunk 3 above)
Comment (Nikoleta-v3) [reply main.tex]: Yup that sounds better 👍

File: main.tex (hunk 2 above)
Comment (drvinceknight) [reply main.tex]: I think I'd go with:


```

more specifically, \textit{memory-one best responses}. It was shown that:

\item The utility...
\item There is a...
```

Comment (Nikoleta-v3) [reply main.tex]: `There is a compact way of identifying a memory-one best response...` this was also proven right?

There is theorem proven this. But I don't mind shown either 👍

File: main.tex [hunk 4]
```diff
@@ -245,20 +244,11 @@ \section{Methods}
 and restricted memory.
 
 \section{Results}\label{section:results}
-% \subsection{Best response memory-one strategies}\label{section:best_response_memory_one}
 
-As discussed ZDs have been acclaimed for their robustness
-against a single opponent. ZDs are evidence that extortion works
+ZDs are evidence that extortion works
 in pairwise interactions, their behaviour ensures that the strategies will never
```
Comment (Nikoleta-v3) [inline main.tex]: This sentence feels a bit standalone. 

> ZDs are evidence that extortion works
in pairwise interactions, their behaviour ensures that the strategies will never
lose a game.

> Now we are going to define best response memory-one strategies

File: main.tex [hunk 5]
```diff
@@ -289,19 +279,26 @@ \section{Results}\label{section:results}
...
-Self interactions can be incorporated in the
-formulation that has been used so far. More specifically, the optimisation
-problem of Eq.~\ref{eq:mo_tournament_optimisation} is extended to:
+In multi opponent interactions,
+where the payoffs matter, strategies trying to exploit their opponents will
+suffer.
+Compared to ZDs, best response memory-one strategies, which have a
+theory of mind of their opponents, utilise their behaviour in order to gain the
+most from their interactions. The question that arises then is whether best
+response strategies are optimal because they behave in an extortionate way.
```
Comment (Nikoleta-v3) [inline main.tex]: Can we check that we are okay with how these two paragraphs follow each other?

File: main.tex (hunk 4 above)
Comment (drvinceknight) [reply main.tex]: I think I agree, I've removed it.

**Comment by Nikoleta-v3** [comment]
👍  I am gonna go ahead an merge now.

**Pull request merged**